scripts/revert_upgrade_voting_delegation_holesky.py

```diff
@@ -0,0 +1,92 @@
+"""
+Revert Voting Delegation Upgrade
```

Comment:
Let's add "[in case of emergency]" as in the script "revert_vote_2022_06_21"

Reply:
Fixed in 98c41312c4dcbf45286acc799ee16c3e46be578a